skyuxconfig-schema.json

```diff
@@ -279,6 +279,10 @@
       "description": "Specifies a path to reference SKY UX components in Angular. This property is specific to the SKY UX docs site and is for internal Blackbaud use only.",
       "type": "string"
     },
+    "incompatibleDependencies": {
```

Comment:
Do we need a docs issue to add this new config property to the reference docs at https://developer.blackbaud.com/skyux/learn/reference/configuration?

Also, as I was looking through https://github.com/blackbaud/skyux-config/blob/336a06a422249169533f00be4f930a0ae527c0a3/skyuxconfig-schema.json to check out this new property, I noticed a few other things:
- For the `app` config property, do we need to create a docs issue to add the `theming` option at https://github.com/blackbaud/skyux-config/blob/336a06a422249169533f00be4f930a0ae527c0a3/skyuxconfig-schema.json#L126?
- The `pacts` options that we document at https://developer.blackbaud.com/skyux/learn/reference/configuration#pacts are not included in the config schema at https://github.com/blackbaud/skyux-config/blob/336a06a422249169533f00be4f930a0ae527c0a3/skyuxconfig-schema.json#L307. Is that a problem?
- The `routes` property at https://github.com/blackbaud/skyux-config/blob/336a06a422249169533f00be4f930a0ae527c0a3/skyuxconfig-schema.json#L351 lists `public` and `referenced` properties, but we don't call them out at https://developer.blackbaud.com/skyux/learn/reference/configuration#routes. Do we need a docs issue to update that?
- The `useHashRouting` property at https://github.com/blackbaud/skyux-config/blob/336a06a422249169533f00be4f930a0ae527c0a3/skyuxconfig-schema.json#L365 is not in alphabetical order like most of the other config properties. Should it be?
- Do we need a docs issue to add the `skyuxModules` property at https://github.com/blackbaud/skyux-config/blob/336a06a422249169533f00be4f930a0ae527c0a3/skyuxconfig-schema.json#L369 to the docs at https://developer.blackbaud.com/skyux/learn/reference/configuration?
Do we need a docs issue to add the `librarySettings` property at https://github.com/blackbaud/skyux-config/blob/336a06a422249169533f00be4f930a0ae527c0a3/skyuxconfig-schema.json#L373 to the docs at https://developer.blackbaud.com/skyux/learn/reference/configuration?

Reply:
I think the only one that _might_ need attention is the `theming` property, but that would be up to Paul/Todd to decide. 
I'll create another PR to remove `skyuxModules` (it's an old property we should have removed during the SKY UX 4 migration), and reorder the `useHashRouting` property.